src/services/upper-bar-manager/models/icon-dto.ts

```diff
@@ -9,7 +9,9 @@ export interface IconDto {
    * has to be corresponds to the names of the icons (plugins) names in the
    * ui-managers config (under upperBarManager.pluginsIconsOrder)
    */
-  label: KalturaPluginNames | string;
+  displayName: string;
```

Comment:
add an explanation about the separate fields

Reply:
done